ownCloud/Theming/UI/ThemeRoundedButton.swift

```diff
@@ -22,20 +22,14 @@ class ThemeRoundedButton: ThemeButton {
 
 	override init(frame: CGRect) {
 		super.init(frame: frame)
-		styleButton()
+
+		self.buttonCornerRadius = -1
```

Comment:
@hosy seen in `ThemeButton` class that negative buttonCornerRadius has a special meaning -> it causes `layer.cornerRadius `to be `bounds.size.height / 2`. Can we may be replace it with some static constant with a good name?

Reply:
Yes, makes sense.

Reply:
I added new enum values to replace the existing values with new enum values